docs/src/appendix/sncast/account/add.md

```diff
@@ -4,7 +4,7 @@ Import an account to accounts file.
 Account information will be saved to the file specified by `--accounts-file` argument,
 which is `~/.starknet_accounts/starknet_open_zeppelin_accounts.json` by default.
 
-## Required Common Arguments — Passed By CLI or Specified in `Scarb.toml`
+## Required Common Arguments — Passed By CLI or Specified in `snfoundry.toml`
 
 * [`url`](../common.md#--url--u-rpc_url)
```

Comment:
Can we consider including this information also for other commands or maybe even in `common.md`? As far as I know `script init` is the only command that currently doesn't require url.

Reply:
sure, created an issue and will handle in separate pr
https://github.com/foundry-rs/starknet-foundry/issues/1759